### PR TITLE
Document excluded namespace configuration only available after v0.8

### PR DESCRIPTION
**Pull request by yiqigao217**

### CODE REVIEW

File: incubator/hnc/docs/user-guide/concepts.md [hunk 1]
```diff
@@ -533,6 +533,8 @@ We are considering replacing this with the standard
 
 #### hnc.x-k8s.io/excluded-namespace (label on namespaces)
 
+***Excluded namespaces configuration is only available in HNC v0.8 and later***
```
Comment (adrianludwin) [inline incubator/hnc/docs/user-guide/concepts.md:536]: I think italics alone would have been fine but I won't hold this up :) Feel free to remove the hold if you like.

**Review by adrianludwin — COMMENTED** [review]
/approve
/lgtm
/hold

Feel free to remove the hold if you like the bold italics. To my eyes, it looks a bit too strong (e.g. stronger than the L4 header) and regular italics would be fine but up to you

File: incubator/hnc/docs/user-guide/concepts.md (hunk 1 above)
Comment (yiqigao217) [reply incubator/hnc/docs/user-guide/concepts.md:536]: I'm using the same format as https://github.com/kubernetes-sigs/multi-tenancy/blob/master/incubator/hnc/docs/user-guide/concepts.md#exceptions-and-propagation-control in the same page. So I will remove the hold :)

**Comment by yiqigao217** [comment]
/hold cancel

FYI using the same format as https://github.com/kubernetes-sigs/multi-tenancy/blob/master/incubator/hnc/docs/user-guide/concepts.md#exceptions-and-propagation-control on the same page.

**Comment by adrianludwin** [comment]
sgtm, thanks

**Pull request merged**